tests/integration/test.sync.js

```diff
@@ -332,11 +332,11 @@ adapters.forEach(function (adapters) {
       replications.cancel();
     });
 
-    it.skip('Test sync cancel called twice', function (done) {
+    it('Test sync cancel called twice', function (done) {
       var db = new PouchDB(dbs.name);
       var remote = new PouchDB(dbs.remote);
       var replications = db.sync(remote).on('complete', function () {
-        setTimeout(done); // let cancel() get called twice before finishing
+        done();
```

Comment:
The test preceding it is also skipped, https://github.com/pouchdb/pouchdb/issues/4679 was the issue and it was super hard to find, I dont want to reenable these tests without being sure that particular issue is fixed (the reported error rate was around 1 in 15 test runs). The setTimeout I am less worried about